contrib/build-container.sh

```
@@ -119,6 +119,14 @@ get_options() {
                 missing_requirement $1
             fi
             ;;
+        --torch-versions)
+            if [ "$2" ]; then
+                WHL_TORCH_VERSIONS=$2
+                shift
+            else
+                missing_requirement $1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)